LAYOUT_STYLE_GUIDE.md

```diff
@@ -141,13 +151,15 @@ POST-ASSIGNMENT SECTION CONTENT.
 
 1. `### Introduction`: A brief summary on what the project is and an overview of what the user will be building. Replace the `A BRIEF INTRODUCTION.` text with your own project introduction.
 
-2. `### PRE-ASSIGNMENT SECTION HEADING`: *Optional*. A section that contains content that should come before the actual project assignment. This section will most likely not be needed for most projects, but when it is needed simply replace the `PRE-ASSIGNMENT SECTION HEADING` text with a proper section heading and the `PRE-ASSIGNMENT SECTION CONTENT.` text with your own content. Then add any additional pre-assignment sections. **If the project does not have a pre-assignment section, remove this entire section from the project.**
+1. `### PRE-ASSIGNMENT SECTION HEADING`: *Optional*. A section that contains content that should come before the actual project assignment. This section will most likely not be needed for most projects, but when it is needed simply replace the `PRE-ASSIGNMENT SECTION HEADING` text with a proper section heading and the `PRE-ASSIGNMENT SECTION CONTENT.` text with your own content. Then add any additional pre-assignment sections. **If the project does not have a pre-assignment section, remove this entire section from the project.**
+
+1. `### Assignment`: A numbered list of items that describe detailed requirements or user stories that must be followed in order to complete the project. Replace the `A REQUIREMENT/USER STORY.` with your own requirement, then add any additional numbered requirement items.
 
-3. `### Assignment`: A numbered list of items that describe detailed requirements or user stories that must be followed in order to complete the project. Replace the `A REQUIREMENT/USER STORY.` with your own requirement, then add any additional numbered requirement items.
+1. `#### CUSTOM ASSIGNMENT HEADING`: *Optional*. A heading that describes the assignment content that follows. This can be useful for breaking up an assignment into multiple lists of instructions.
```

Comment:
```suggestion
1. `#### CUSTOM ASSIGNMENT HEADING`: *Optional*. A heading that describes the assignment content that follows. This can be useful for breaking up an assignment into multiple lists of instructions. If an assignment is intended to have multiple lists, each list should include a custom heading, otherwise this custom heading can be omitted.
```

Not sure if this should be added just as it is in the lesson template?

Reply:
True. Maybe instead of including the assignment heading text in the template we just mention inline in the "Assignment" description that headings can be added?

Reply:
That could work. Either way would be more in line with how the lesson template describes it.

Reply:
Just to be sure, the suggestion I made was for `line 151` which is in the `project layout` explanation  to be more in line with the `lesson layout` explanation. I se your made edits to the `lesson layout` description of this.